test/e2e/step-definitions/backtooffice-steps.ts

```diff
@@ -1,6 +1,6 @@
 import { Then, When } from 'cucumber';
-import { getElement, clickElement } from './generic-steps';
 import { browser, by, ExpectedConditions } from 'protractor';
+import { getElement, clickElement  } from '../../helpers/helpers';
```

Comment:
Don't like the name of helpers nested within a folder called helpers... would have been better to rename the file to something indicating what type of helpers they are - e.g interactionHelpers or domHelpers.... but I know it wasn't changed in this PR.